theia-extensions/viewer-prototype/src/browser/trace-overview-preference.ts

```diff
@@ -0,0 +1,33 @@
+import { PreferenceProxy, PreferenceSchema, PreferenceScope } from '@theia/core/lib/browser';
+
+export const TRACE_OVERVIEW_DEFAULT_VIEW = 'trace Viewer.trace Overview.defaultView';
+
+export function getSwitchToDefaultViewErrorMessage(preferredName: string, defaultName: string): string {
+    return `The ${preferredName} view cannot be opened as the trace overview. ` +
+    `Opening ${defaultName} instead. ` +
+    'Please set the Default View Preference (Ctrl + ,) of the Trace Overview to an XY view, ' +
+    'or make sure the name is spelled correctly. ';
+}
+
+export function getOpenTraceOverviewFailErrorMessage(): string {
+    return 'An error has occurred while opening the trace overview.';
+}
+
+export const OverviewSchema: PreferenceSchema = {
+    scope: PreferenceScope.Folder,
+    type: 'object',
+    properties: {
+        [TRACE_OVERVIEW_DEFAULT_VIEW]: {
+            type: 'string',
+            description: 'Specify the name of the view that will be used as the default view for the Trace Overview. ' +
```

Comment:
set the `default` value to histogram here:

Reply:
Wouldn't this make the default value a hard coded value? Would it still work with different server builds that do not have the Histogram view?

Reply:
Not really. When building a Theia base application, you can specify configure the default in the package.json of that application.

Reply:
Marking as Unresolved to track the missing package.json how-to part, as discussed yesterday.